style.css

```diff
@@ -326,3 +326,259 @@ hr {
   height: 29px;
   left: 76px;
 }
+
+@media only screen and (min-width: 768px) {
```

Comment:
- [ ] kindly review that the mobile version still working as expected. 👀 

![image](https://user-images.githubusercontent.com/58642949/187030182-73c43f94-bb91-4020-956b-0a92d7bad8f8.png)